docs/conversion.md

```diff
@@ -13,7 +13,7 @@ This document outlines all the conversion details regarding `docker-compose.yaml
 | depends_on |  | N |  |  |
 | dns |  | N |  |  |
 | dns_search |  | N |  |  |
-| tmpfs |  | N |  |  |
+| tmpfs |  | Y |  | Creates `emptyDir` volume & mounts inside container | 
```

Comment:
Description is only partially true.
This descriptions makes it look like its regular `emptyDir ` volume.
That is not true. Its `emptDir` with   `medium: Memory`,  it actually mounts given dir as tmpfs inside container.